### PR TITLE
Test executeGraph helper method

### PR DESCRIPTION
**Pull request by landim**

### CODE REVIEW

File: src/Dse.Graph.Test.Integration/StatementIntegrationTest.cs [hunk 1]
```diff
@@ -21,14 +21,31 @@ public class StatementIntegrationTest : BaseIntegrationTest
...
+            // The result should be DSE driver vertices
+            VerifyGraphResultSet(rs);
+            VerifyGraphResultSet(rsSync);
+        }
+
+        private static void VerifyGraphResultSet(GraphResultSet rs)
+        {
             var v = rs.First().ToVertex();
             Assert.AreEqual("person", v.Label);
             Assert.AreEqual("marko", v.Properties["name"].ToArray()[0].Get<string>("value"));
```
Comment (jorgebay) [inline src/Dse.Graph.Test.Integration/StatementIntegrationTest.cs]: We should replace the usage of properties with the new `GetProperty()` method.

**Comment by jorgebay** [comment]
+1 for the extra coverage!
The Jenkins job didn't kick in though... :(

**Comment by landim** [comment]
Addressed the usage of GetProperty method

**Comment by jorgebay** [comment]
Do u have an idea why the Jenkins job didn't kicked in?

It was working for #7.

**Comment by landim** [comment]
I will rename the branch to avoid naming issues on jenkins

**Pull request closed without merging**